### PR TITLE
Move Scenarioo Tutorial to Docs

### PR DESCRIPTION
**Pull request by bruderol**
See #629

* Move Scenarioo Tutorial form webpage to docs
* Link appropriately in menu

### CODE REVIEW

**Comment by bruderol** [comment]
@ideco I would recommend to create a new folder `tutorial` (at top level, next to `setup`) for our tutorial and make sections (several md files) inside this folder. Then don't forget to put this sections into `SUMMARY.md` in `docs` --> will appear in the menu. Also I would use a README.md in the `tutorial` with the first overview chapter, explaining major concept, what scenarioo is, how to download the libraries etc. (with links to other pages for mor edetailed explanations) etc. - ok?

**Comment by bruderol** [comment]
As discussed right now:
* Name the new page "Use Scenarioo Writer in Tests"
* link in SUMMARY.md
* remove the tutorial directory I just added and other links in SUMMARY.md I added (sorry for that)
* Maybe merge it with http://scenarioo.org/docs/setup/How-to-use-Scenarioo-Writer-Library.html
* Also adjust the "Setup" root introduction Page and maybe rename this section to "Getting Started" ?
* Dont forget to adjust all links to old Tutorial, e.g. like here: https://github.com/scenarioo/scenarioo-java/blob/master/docs/usage.md#tutorial

Later:
Cleanup `setup` directory - I think we should separate this into `tutorial` (="getting Started") and have all advanced features (e.g. like advanced configuration possibilites) into the `features` directory, where we habe other short tutorials about how to use specific advanced features (e.g. like the generic object repository or diff viewer, full text search, ScreenAnnotations, Labels, etc. etc.)

**Comment by bruderol** [comment]
I adjusted #629 as discussed - thanks

**Comment by bruderol** [comment]
@ideco we have to merge with the release branch were I started to adjust some docu that you moved now :-(

**Comment by ideco** [comment]
Dang! I didn't see the feature branch :-)

**Comment by bruderol** [comment]
@ideco I roughly went through the high level pages in the new tutorial section and adjusted some things regarding the writer library pages - we could probably still simplify that further, but for a start it should be ok.

What is missing:
* we need to adjust any links in any of our other repos: Java, Js, C#, Webpage!

**Comment by bruderol** [comment]
I will try to get this finished for the release this morning.

@adiherzog please review as well and give me feedback if you see something important - thanks!

File: docs/SUMMARY.md [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
-  * [Migration Guide](setup/Migration-Guide.md)
+* [Getting Started](tutorial/README.md)
+  * [Install Viewer Web App](tutorial/Scenarioo-Viewer-Web-Application-Setup.md)
+  * [Write Reports from Tests](tutorial/How-to-use-Scenarioo-Writer-Library.md)
+     * [Quickstart with Scenarioo-JS](tutorial/Quickstart-JS.md)
+     * [Use Scenarioo in Java Selenium Tests](tutorial/Using-Scenarioo-In-Java-Selenium-Tests.md)
+  * [Install Viewer Docker Image](tutorial/Scenarioo-Viewer-Docker-Image.md)
+  * [Publish Reports](tutorial/Publish-Documentation-Data.md)
+  * [Viewer Configuration](tutorial/Configuration.md)
+  * [Migration Guide](tutorial/Migration-Guide.md)
```
Comment (adiherzog) [inline docs/SUMMARY.md]: Migration Guide should not be under "Getting Started", as this is only interesting for people that have been using Scenarioo for a while.

Comment (adiherzog) [reply docs/SUMMARY.md]: Move to changelog

**Comment by bruderol** [comment]
@adiherzog we improved a lot in the docu - I still think that there are many things to improve but we will now start tomerge that and to release - ifd you see something later we can still improve.

**Pull request merged**